src/main/java/org/wildfly/security/asn1/ASN1Encoder.java

```diff
@@ -171,6 +172,17 @@
     void encodeBitString(String binaryStr);
 
     /**
+     * Encode an ASN.1 bit string where the value represents the binary form of the given <code>integer</code>.
+     *
+     * @param integer the integer to encode as a binary string
+     */
+    default void encodeBitString(BigInteger integer) {
+        ByteStringBuilder target = new ByteStringBuilder();
+        new DEREncoder(target).encode(integer);
```

Comment:
DEREncoder shouldn't be referenced from ASN1Encoder - the implementation of this method should just be moved to DEREncoder.